examples/hat/binaryclock.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import unicornhat as uh
+import datetime
+from time import sleep
+
+print("""Binar Clock made by Iorga Dragos Florian
+Displays Hour(red), Minute(yellow), Second(blue), Day(green) and Month(Pink) in binary format""")
+FPS = 30.0
+uh.rotation(0)
+uh.brightness(1)
+
+def draw_time(b,x,idx,color):
+    if b == '1':
+        uh.set_pixel(x,idx,color)
+    sleep(1.0/FPS)
+
+while True:
+    now = datetime.datetime.now()
+    month = bin(now.month)[2:][::-1]
```

Comment:
As suggested- I think this should be simplified to `month = now.month` and the rest moved into `draw_time`.